Police-Data-Accessibility-Project/data-sources-app-v2
# feature: search results first iteration

This page and its design is still a work in progress. This PR gets the basic search results functionality down. We'll fine-tune it further later.

## To test
1. run the app locally
2. Search
3. Observe search results page
